language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/rangeformat/RangeFormattingTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Is this a new test? Or moving an existing one?

Reply:
This is a new test. It's a mistake that the header contains an old year. I'll rectify this

Reply:
Fixed

Reply:
Fixed